### PR TITLE
introduce CI

### PR DESCRIPTION
**Pull request by janek26**
adds semantic commit as a pre-commit hook and semantic release to create releases out of these commit messages

### CODE REVIEW

**Comment by avimak** [comment]
@yoga-braavos , pls review 👍

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # get-starknet<br/>StarkNet wallet <-> dApp bridge
 
+[![npm](https://img.shields.io/npm/v/get-starknet-wallet.svg)](https://www.npmjs.com/package/get-starknet-wallet)
```
Comment (avimak) [inline README.md:3]: @janek26 wrong npm name 👆

Comment (janek26) [reply README.md:3]: good catch

File: .releaserc [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            {
+                "assets": [
+                    "package.json",
+                    "package-lock.json",
+                    "CHANGELOG.md"
+                ],
+                "message": "chore(release): ${nextRelease.version} [skip ci]\n\n${nextRelease.notes}"
+            }
+        ],
+        "@semantic-release/github"
```
Comment (yoga-braavos) [inline .releaserc:25]: Shouldn't we define `assets` explicitly?

File: .releaserc [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        {
+            "name": "develop",
+            "channel": "next"
+        }
+    ],
+    "plugins": [
+        "@semantic-release/commit-analyzer",
+        "@semantic-release/release-notes-generator",
+        "@semantic-release/changelog",
+        "@semantic-release/npm",
```
Comment (yoga-braavos) [inline .releaserc:13]: Shouldn't we define `assets` explicitly?

**Comment by janek26** [comment]
@yoga-braavos ghanks for the fix! I think not having assets in the github and npm module is fine for now, ready for merge if you agree.

**Review by yoga-braavos — APPROVED** [review]

**Pull request merged**